playbooks/support/scheduling.md

```diff
@@ -65,6 +65,21 @@ rotations start/end on a Monday at 11am. "B" rotations start/end on a Wednesday
 
 [team navigator]: https://team.artsy.net
 
```

Comment:
Will add the setup steps in a separate PR just wanted to get the volunteer steps up asap!